src/launchpad/size/insights/common.py

```diff
@@ -97,3 +98,40 @@ def generate(self, input: InsightsInput) -> LargeVideoFileInsightResult:
         total_savings = sum(file.size // 2 for file in large_files)
 
         return LargeVideoFileInsightResult(files=large_files, total_savings=total_savings)
+
+
+class LargeAudioFileInsight(Insight[LargeAudioFileInsightResult]):
+    """Insight for identifying audio files larger than 5MB."""
+
+    def generate(self, input: InsightsInput) -> LargeAudioFileInsightResult:
+        size_threshold_bytes = 5 * 1024 * 1024  # 5MB
```

Comment:
Assuming just arbitrary right now? Fine with this number, might be good to just leave a comment that this was chosen arbitrarily.